packages/trie/examples/merkle_patricia_trees/README.md

```diff
@@ -209,145 +205,129 @@ Creating a branch node is a bit more complicated. For a branch to exist, we need
+      101,  65
+    ]
+  ],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) [],
+  Uint8Array(0) []
 ]
 ```
 
-14/16 of its branches are empty (`<Buffer >`). Only the branches at indexes `3` and `4` contain values. Now here's an important point. These indexes are not determined at random: **they correspond to the next hex-value of the path of our two keys (hex values `3` and `4`)**. This makes sense, as this is the value at which our keys diverge:
+14/16 of its branches are empty (`Uint8Array`s). Only the branches at indexes `3` and `4` contain values. Now here's an important point. These indexes are not determined at random: **they correspond to the next hex-value of the path of our two keys (hex values `3` and `4`)**. This makes sense, as this is the value at which our keys diverge (converted to hex for easier comparison):
 
 ```jsx
-//      <------ same ------> <-> (different)
-<Buffer 74 65 73 74 4b 65 79 30>           // "testKey0", "0" = 30 in bytes
-<Buffer 74 65 73 74 4b 65 79 41>           // "testKeyA", "A" = 41 in bytes
+// <---- same----> <-> (different)
+0x7465737456616c756530 // "testKey0", "0" = 30 in bytes
+0x7465737456616c756541 // "testKeyA", "A" = 41 in bytes
 ```
 
-Going back to the branches above, we see that our two branches (at index `3` and `4`) are:
+Going back to the branches above, we see that our two branches (at index `3` and `4`) are (converted to hex again for better readability):
+
+Node 1 branch 3 (hex): 0x30 - 0x7465737456616c756530
+Node 1 branch 4 (hex): 0x31 - 0x7465737456616c756541
 
 ```jsx
-// <--path-->  <------------- value ----------------->
-[ <Buffer 30>, <Buffer 74 65 73 74 56 61 6c 75 65 30> ] // going down the path of "testKey0"
-[ <Buffer 31>, <Buffer 74 65 73 74 56 61 6c 75 65 41> ] // going down the path of "testKeyA"
+[path:  0x30  | value:  0x7465737456616c756530] // going down the path of "testKey0"
+[path:  0x31  | value:  0x7465737456616c756541] // going down the path of "testKeyA"
 ```
 
-I'm getting ahead of myself here, but here's a spoiler: these are leaf nodes. We see that each line contains two "buffers" (buffers are sequences of bytes). And indeed: leaf nodes are arrays containing two items. The first item is the remaining path (we'll get back to what the hex values stand for in a minute), while the second buffer is the value at that path. We can confirm that the values are what we'd expect:
+I'm getting ahead of myself here, but here's a spoiler: these are leaf nodes. We see that each line contains two "bytes" (bytes are sequences of bytes). And indeed: leaf nodes are arrays containing two items. The first item is the remaining path (we'll get back to what the hex values stand for in a minute), while the second bytes array is the value at that path. We can confirm that the values are what we'd expect:
```

Comment:
I think "bytes are sequences of bytes" can be confusing to understand. Maybe rewording it would work better.
```suggestion
I'm getting ahead of myself here, but here's a spoiler: these are leaf nodes. We see that each line contains two byte arrays (byte arrays are sequences of bytes). And indeed: leaf nodes are arrays containing two items. The first item is the remaining path (we'll get back to what the hex values stand for in a minute), while the second bytes array is the value at that path. We can confirm that the values are what we'd expect:
```